go.mod

```diff
@@ -3,16 +3,16 @@ module sigs.k8s.io/sig-storage-lib-external-provisioner/v5
 go 1.13
 
 require (
-	github.com/miekg/dns v1.1.27
+	github.com/miekg/dns v1.1.29
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
-	github.com/prometheus/client_golang v1.4.1
+	github.com/prometheus/client_golang v1.5.1
 	github.com/prometheus/client_model v0.2.0
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
-	k8s.io/api v0.17.3
-	k8s.io/apimachinery v0.17.3
+	k8s.io/api v0.18.0
+	k8s.io/apimachinery v0.18.0
 	k8s.io/client-go v0.0.0-20200131194156-19522ff28802 // release-14.0
```

Comment:
here also could use an update to v0.18.0

Reply:
Done.